admin/app/model/AdminLifecycle.scala

```diff
@@ -89,13 +89,15 @@ class AdminLifecycle(appLifecycle: ApplicationLifecycle,
       matchDayRecorder.record()
     }
 
-    if (environment.isProd) {
```

Comment:
This check was unnecessary since the email job itself relies on an `email: Option[String]` which is only configured in `PROD`